packages/experience-builder-sdk/src/ExperienceRoot.tsx

```diff
@@ -35,8 +25,7 @@ export const ExperienceRoot = ({
   });
 
   if (isEditorMode) {
-    const entityStore =
-      experience && !isDeprecatedExperience(experience) ? experience.entityStore : undefined;
+    const entityStore = experience ? experience.entityStore : undefined;
```

Comment:
Maybe this line can be simplified even more now
```suggestion
    const entityStore = experience?.entityStore;
```